lib/utils.pm

```diff
@@ -239,6 +240,10 @@ sub is_jeos() {
     return get_var('FLAVOR', '') =~ /^JeOS/;
 }
 
+sub is_casp() {
```

Comment:
as long as `is_casp` is only used in the sle main.pm I recommend to also define the sub there (trying to always define the code to the nearest point where it is needed while avoiding duplication).